packages/workflow/package.json

```diff
@@ -58,7 +58,7 @@
     "lodash.merge": "^4.6.2",
     "lodash.set": "^4.3.2",
     "luxon": "^2.3.0",
-    "riot-tmpl": "github:n8n-io/tmpl",
+    "riot-tmpl": "npm:@n8n_io/riot-tmpl@^1.0.1",
```

Comment:
Why not as usual?
```suggestion
    "@n8n_io/riot-tmpl": "^1.0.1",
```

Reply:
Yeah, good point. We won't revert back to riot-tmpl. Updated imports.